pages/index.js

```diff
@@ -30,12 +47,16 @@ export default function Home({ locale }) {
         <h1 className="text-4xl text-bold">{t("findSupport")}</h1>
       </div>
 
-      <section className="">
+      <section id="popular_catagories" className="">
         <h3 className="text-2xl text-bold py-3">{t("popularCatagories")}</h3>
-        <p>{t("underConstruction")}</p>
```

Comment:
t, translate no longer serves a purpose in this file either

Reply:
Does it not? I see 3 other places that it's being used.

Reply:
I might be legally blind